src/main/java/net/mcreator/workspace/ShareableZIPManager.java

```diff
@@ -41,6 +41,32 @@ public class ShareableZIPManager {
 
 	private static final Logger LOG = LogManager.getLogger("Shareable ZIP manager");
 
+	private static final Set<String> EXCLUDES = new HashSet<>() {{

```

Comment:
This will be formatted by IDEA. Either use IDEAs formatting or wrap with formatted off/on comments

Reply:
I tried running the formatter in this file and the only line affected was this:
```diff
-	public record ImportResult(File file, boolean regenerateRequired){}
+	public record ImportResult(File file, boolean regenerateRequired) {}
```

Reply:
Ah you are right :D